src/api/common/utils.js

```diff
@@ -20,8 +20,20 @@ function convertAmount(amount) {
   };
 }
 
+function wrapCatch(asyncFunction: any): any {
```

Comment:
Yeah, but this one isn't worth the effort unless you want to try to figure it out.